.golangci.yml

```diff
@@ -139,7 +139,7 @@ linters:
     - errcheck
     - funlen
     - gochecknoinits
-    - goconst
+    # - goconst
```

Comment:
Can we disable it only for one file where we're using the default constants? See at example https://github.com/networkservicemesh/sdk/blob/main/.golangci.yml#L170

Reply:
Updated 👍